app/controllers/wikis_controller.rb

```diff
@@ -1,8 +1,72 @@
-# This controller will handle our about and index views and log us in
+# This controller will trigger login, and handle our wiki CRUD methods
 class WikisController < ApplicationController
   before_action :authenticate_user!
+  before_action :create_new_wiki, only: [:new, :create]
```

Comment:
I am generally not a fan of callbacks for things like this -- it is not always immediately clear where instance variables are coming from when reading an action in isolation. I know you're trying to DRY up your code here and for that I commend you.